src/Dashboard/Filter.php

```diff
@@ -34,19 +34,18 @@
  */
 
 namespace Glpi\Dashboard;
-
 use Glpi\Plugin\Hooks;
-use Group;
-use Html;
-use ITILCategory;
-use Location;
-use Manufacturer;
-use Plugin;
-use RequestType;
 use Session;
-use State;
-use Ticket;
-use User;
+use Glpi\Dashboard\Filters\DatesFilter;
+use Glpi\Dashboard\Filters\ItilCategoryFilter;
+use Glpi\Dashboard\Filters\LocationFilter;
+use Glpi\Dashboard\Filters\ManufacturerFilter;
+use Glpi\Dashboard\Filters\RequestTypeFilter;
+use Glpi\Dashboard\Filters\StateFilter;
+use Glpi\Dashboard\Filters\TicketTypeFilter;
+use Glpi\Dashboard\Filters\GroupTechFilter;
+use Glpi\Dashboard\Filters\UserTechFilter;
+use Glpi\Dashboard\Filters\DatesModFilter;
```

Comment:
```suggestion
use Glpi\Dashboard\Filters\{
    DatesFilter,
    ItilCategoryFilter,
    LocationFilter,
    ManufacturerFilter,
    RequestTypeFilter,
    StateFilter,
    TicketTypeFilter,
    GroupTechFilter,
    UserTechFilter,
    DatesModFilter
};
```

Not a mandatory change, but fyi, you can have a shorter syntax for importing several classes of the same namespace)